protocols/imap/src/main/java/org/apache/james/imap/processor/AbstractMessageRangeProcessor.java

```diff
@@ -84,6 +85,11 @@ protected Mono<Void> processRequestReactive(R request, ImapSession session, Resp
                 return ReactorUtils.logAsMono(() -> LOGGER.debug("{} failed from mailbox {} to {} for invalid sequence-set {}",
                     getOperationName(), session.getSelected().getMailboxId(), targetMailbox, request.getIdSet(), e));
             })
+            .onErrorResume(OverQuotaException.class, e -> {
+                taggedBad(request, responder, HumanReadableText.FAILURE_OVERQUOTA);
```

Comment:
Should be a no, should have OVERQUOTA response code.

Reply:
Updated